tests/test_queries.py

```diff
@@ -218,3 +218,42 @@ def test_mapping_resolver_to_object_attribute():
     result = graphql(schema, "{ user { firstName } }")
     assert result.errors is None
     assert result.data == {"user": {"firstName": "Joe"}}
+
+
+def test_accept_list_of_resolvers_map():
+    type_defs = """
+        type Query {
+            user: User
+            test(data: TestInput): Int
+        }
+
+        type User {
+            firstName: String
+        }
+
+        input TestInput {
+            value: Int
+        }
+    """
+
+    def resolve_test(*_, data):
+        assert data == {"value": 4}
+        return "42"
+
+    resolvers = [
+        {
+            "Query": {"user": lambda *_: Mock(first_name="Joe")},
+            "User": {"firstName": resolve_to("first_name")},
+        },
+        {"Query": {"test": resolve_test}},
```

Comment:
What happens when later resolver map redefines the resolver defined in previous one? We'll need to know this and have tests harness in place (just like how we are testing merging typedefs in `test_modularization`).

For reference, here's [`lodash.merge`](https://lodash.com/docs/4.17.10#merge) mechanism used by Apollo-Server:

> It recursively merges own and inherited enumerable string keyed properties of source objects into the destination object. Source properties that resolve to undefined are skipped if a destination value exists. Array and plain object properties are merged recursively. Other objects and value types are overridden by assignment. Source objects are applied from left to right. Subsequent sources overwrite property assignments of previous sources.